modules/nw-secondary-ext-gw-configure.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-external-gateway.adoc
+
+:_content-type: PROCEDURE
+[id="nw-secondary-ext-gw-configure_{context}"]
+= Configure a secondary external gateway
+
+You can configure a secondary external gateway for a namespace in your cluster.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a YAML file that contains an `AdminPolicyBasedExternalRoute` object.
+. To create an admin policy based external route, enter the following command:
++
+[source,terminal]
+----
+$ oc create -f <file>.yaml
+----
++
+--
+where:
+
+`<file>`:: Specifies the name of the YAML file that you created in the previous step.
+--
++
+.Example output
+[source,text]
+----
+adminpolicybasedexternalroute.k8s.ovn.org/default-route-policy created
+----
+
+. To confirm that the admin policy based external route was created, enter the following command:
++
+[source,terminal]
+----
+$ oc describe apbexternalroute <name> | tail -n 6
+----
++
+--
+where:
+
+`<name>`:: Specifies the name of the `AdminPolicyBasedExternalRoute` object.
+--
++
+.Example output
+[source,text]
+----
+Status:
+  Last Transition Time:  2023-04-24T15:09:01Z
+  Messages:
+  Configured external gateway IPs: 172.18.0.8
+  Status:  Success
+Events:  <none>
+----
+
+.Verification
+
+If you created an `AdminPolicyBasedExternalRoute` object that selects a host-network pod IP address as the secondary external gateway, you can confirm that the next hop is correct for a pod with the following steps:
+
+. To get the IP address of the pod, enter the following command:
++
+[source,terminal]
+----
+oc get pods/<pod_name> -n <namespace> -o wide
+----
++
+--
+where:
+
+`<pod_name>`:: Specifies the name of the pod.
+`<namespace>`:: Specifies the namespace of the pod.
+--
+
+. Confirm that the IP address from the previous step is available as an external gateway.
+
+.. To find the the OVN-Kubernetes control plane pod that manages the next hop for the pod, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app=ovnkube-master -n openshift-ovn-kubernetes | grep <node_name>
```

Comment:
@jab-rh We still need to fix this as mentioned on email thread  "-l app=ovnkube-master"  is OCP 4.13 pre OVNK-IC. For OCP 4.14 we have to use what Jose suggested, which would get you to the correct container running nbdb, where you'd execute the show route command at line 99: 

This in my case:
$ oc get pod dummy1 -n bar -o wide   (Command from L#70)
NAME     READY   STATUS    RESTARTS   AGE   IP             **NODE**         NOMINATED NODE   READINESS GATES
dummy1   1/1     Running   0          16h   10.128.3.229   **worker-0-1**   <none>           <none>

Open a bash session on nbdn container on host worker-0-1 where pod dummy1 is running. 
#POD=$(kubectl get pod -n openshift-ovn-kubernetes --field-selector spec.nodeName=**worker-0-1** -o jsonpath='{range .items[*]}{.[metadata.name](http://metadata.name/)}{"\n"}{end}' | grep ovnkube-node-) ; kubectl exec -ti $POD -n openshift-ovn-kubernetes -c nbdb -- bash

On the bash session started above, execute the show route #ovn-nbctl lr-route-list GR_ovn-work,  I'm not sure but suspect this here should replace L#99.